api/src/main/resources/messages.properties

```diff
@@ -648,4 +648,29 @@ coreapps.conditionui.active.label=ACTIVE
 coreapps.conditionui.inactive.label=INACTIVE
 coreapps.conditionui.undo=Undo delete
 coreapps.conditionui.activeConditions=Active Conditions
-coreapps.conditionui.inactiveConditions=Inactive Conditions
```

Comment:
What change is this?

Reply:
i seem not able to see the actual change , its neither a space nor a new line , cant see the actual change